### PR TITLE
Added "TSG" back to template

### PR DESCRIPTION
**Pull request by ericc1103**
The gallery hotfix is now in PROD so I would like to publish this again.

### CODE REVIEW

File: Workbooks/TSG/Performance Counter TSG/Performance Counter TSG.workbook [hunk 1]
```diff
@@ -0,0 +1,231 @@
+{
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+  "version": "Notebook/1.0",
+  "isLocked": false,
```
Comment (acearun) [inline Workbooks/TSG/Performance Counter TSG/Performance Counter TSG.workbook]: True?

File: Workbooks/TSG/Performance Counter TSG/settings.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Performance Counter TSG",
```
Comment (acearun) [inline Workbooks/TSG/Performance Counter TSG/settings.json]: Performance Counters

File: Workbooks/TSG/Performance Counter TSG/settings.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    "author": "Microsoft",
+    "galleries": [{ "type": "tsg", "resourceType": "microsoft.insights/components", "order": 100 }]
```
Comment (acearun) [inline Workbooks/TSG/Performance Counter TSG/settings.json]: Why not type == 'Troubleshooting Guide'?

Comment (acearun) [inline Workbooks/TSG/Performance Counter TSG/settings.json]: Unrelated to this PR: can I not specify different 'UI groups' for the same template under different galleries?

Comment (ericc1103) [reply Workbooks/TSG/Performance Counter TSG/settings.json]: This is a key that won't be shown to an user

File: Workbooks/TSG/Performance Counter TSG/Performance Counter TSG.workbook (hunk 1 above)
Comment (ericc1103) [reply Workbooks/TSG/Performance Counter TSG/Performance Counter TSG.workbook]: I don't know. Is it? I have copied from the data that saved in shared.

File: Workbooks/TSG/Performance Counter TSG/settings.json (hunk 3 above)
Comment (ericc1103) [reply Workbooks/TSG/Performance Counter TSG/settings.json]: Not sure what you are asking for.. Can we take it offline?

File: Workbooks/TSG/Performance Counter TSG/Performance Counter TSG.workbook (hunk 1 above)
Comment (acearun) [reply Workbooks/TSG/Performance Counter TSG/Performance Counter TSG.workbook]: Should be true - the report opens up locked.

File: Workbooks/TSG/categoryResources.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "en-us": {"name":"Troubleshooting Guides", "description": "", "order": 100}
```
Comment (acearun) [inline Workbooks/TSG/categoryResources.json]: Common Troubleshooting Guided

Comment (ericc1103) [reply Workbooks/TSG/categoryResources.json]: Should it be 'Common Troubleshooting Guide'?

File: Workbooks/TSG/categoryResources.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "en-us": {"name":"Common Troubleshooting Guide", "description": "", "order": 100}
```
Comment (acearun) [inline Workbooks/TSG/categoryResources.json]: Guide**s**

**Pull request merged**